packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/FormRow/formParts/FeatureObservations/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { withRouter } from 'react-router-dom'
+import { Grid } from 'semantic-ui-react'
+
+import config from 'config'
+import createLog from 'utilities/log'
+import { Accordion } from 'coreModules/commonUi/components'
+import { pathBuilder } from 'coreModules/form/higherOrderComponents'
+import { createEnsureAllItemsFetched } from 'coreModules/crud/higherOrderComponents'
+import FeatureObservationsTable from './FeatureObservationsTable'
+import FeatureObservationsTitle from './FeatureObservationsTitle'
+
+const log = createLog(
+  'modules:collectionMammals:MammalForm:SegmentFeatureObservations'
+)
+
+const GROUPS_AND_HEADLINES = [
+  {
+    groups: ['age-and-stage', 'age-stage'],
+    headlineKey: 'age-development-stage',
+  },
+  { groups: ['sex'], headlineKey: 'sex' },
+  { groups: ['condition'], headlineKey: 'condition' },
+  { groups: ['weight'], headlineKey: 'weight' },
+  { groups: ['bone-count'], headlineKey: 'bone-count' },
+  { groups: ['length'], headlineKey: 'length' },
+]
+
+// const ModuleTranslate = createModuleTranslate('collectionMammals')
+
+const propTypes = {
+  allItemsFetched: PropTypes.bool.isRequired,
+  changeFieldValue: PropTypes.func.isRequired,
+  getPath: PropTypes.func.isRequired,
+}
+
+class FeatureObservations extends PureComponent {
+  render() {
+    const {
+      allItemsFetched: allFeatureTypesFetched,
+      changeFieldValue,
+    } = this.props
+    if (!allFeatureTypesFetched) {
+      return null
+    }
+    log.render()
+    return (
+      <Grid.Column width={16}>
+        <Accordion
+          delayItemRenderUntilActive={!config.isTest}
+          fluid
+          items={GROUPS_AND_HEADLINES}
+          renderContent={props => (
+            <FeatureObservationsTable
+              changeFieldValue={changeFieldValue}
+              {...props}
+            />
+          )}
+          renderTitle={props => <FeatureObservationsTitle {...props} />}
+        />
+      </Grid.Column>
+    )
+  }
+}
+
+FeatureObservations.propTypes = propTypes
+
+export default compose(
+  withRouter,
+  createEnsureAllItemsFetched({ resource: 'featureType' }),
+  pathBuilder({ name: 'individual.featureObservations' })
```

Comment:
remove the `name` from here, it should be passed as prop in the part, so it should just be `pathBuilder()`